### ISSUE TITLE
Data Science Topics

### ISSUE THREAD

**Issue by Yog0815** [Yog0815/machine-learning-articles#2]
## TL;DR
Collection of useful data science topics along with code and articles
### Article Link
https://github.com/khuyentran1401/Data-science#data-science-topics
### Author
Khuyen Tran khuyentran1401 
## Key Takeaways
-     Data Science Tools
-     Productive Tools
-     Tools for Deployment
-     Speed-up Tools
-     Math Tools
-     Machine Learning
-     Natural Language Processing
-     Computer Vision
-     Time Series
-     Feature Engineering
-     Visualization
-     Mathematical Programming
-     Scraping
-     Python
-     Terminal
-     Linear Algebra
-     Data Structure
-     Statistics
-     Applications
-     Learning Tips
-     Productive Tips
-     VSCode
-     Book Review
-     Data Science Portfolio
## Useful Code Snippets
```python
#Add your codes here
import numpy as np
import pandas as pd

```

## Useful Tools
* 
*
